core/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -591,6 +591,17 @@ trait DeltaConfigsBase extends DeltaLogging {
     _ == Serializable,
     "must be Serializable"
   )
+
+  /**
+   * Indicates whether row IDs are enabled. When this flag is turned on, all files are guaranteed to
```

Comment:
can you be more specific than `files`? `AddFile`s? `FileAction`s? dataFiles? etc.

Reply:
Changed to `rows`